docs/modules/ROOT/pages/how-tos/add-servala-service-cluster.adoc

```diff
@@ -0,0 +1,46 @@
+= Add a Servala service cluster
+
+Adding a Servala service cluster is a lot less complex than adding a control cluster.
+
+== Add syn configuration
+
+To add a service cluster without a control plane vcluster on top, only a few lines of syn configuration have to be added.
+
+.Syn configs to apply
+[source,yaml]
+----
+classes:
+  - .openshift4 <1>
+
+applications:
+  - appcat <2>
+
+parameters:
+  appcat:
+    clusterManagementSystem:
+      controlPlaneKubeconfig: "?{vaultkv:${cluster:tenant}/${cluster:name}/kubeconfigs/controlclusterID}" <3>
+----
+
+<1> Contains the common configuration for appcat service clusters
+<2> Make sure appcat is imported
+<3> Add the control plane kubeconfig secrets. This might be a common path, if there are more than one service clusters.
```

Comment:
"This might be a common path, if there are more than one service clusters."

what does that mean?